### DISCUSSION TITLE
2022年规划

### DISCUSSION THREAD

**Discussion by liberhe** [Dapp-Learning-DAO/Dapp-Learning#585] [General]
1 分享：
                - 聚焦defi和dao方向[分享](https://docs.qq.com/doc/DQ3hWeVVXQ3JPbXFS)，完成借贷方向的项目（euler）分享， 衍生品和稳定币方向分享开展和跟进（synthetix,  perpetual   protocol , yearn , opyn），DAO方向完成（Gnosis , Augur , DAOstack）项目研究；
                - 邀请更多小伙伴加入DL开源大学进行分享；
                - 每月尝试一次纯英文分享，以及直接邀请项目方（uni,starknet）来做分享；
                - 分享视频剪辑和配上英文字幕。

2 翻译小组： 
                     - 完成mit公开课 后续14集的翻译工作，已翻译到[第10集](https://mp.weixin.qq.com/mp/appmsgalbum?__biz=MzAwNTAxMzMzMA==&action=getalbum&album_id=2241099480056119298#wechat_redirect)
                     - 准备 伯克利defi learning公开课；

3 github仓库（目标2Kstar，目前1.3K star）： 
                         - 完成前60 task以及整理(前30个已完成)；以及gitbook整理，方便开发者阅读；
                         - 项目任务完成defi学习路线图整理（交易-》借贷 -》衍生品 ），方便开发者入门和进阶defi；
                         - 完成密码学以及zkp方向任务, 深入探索zkrollup方向。

4 媒体工具以及国际化（目标1K粉）：
  项目所有链接：
                        - https://linktr.ee/dapplearning 
   媒体工具：
                        -  [twitter](https://twitter.com/Dapp_Learning)
                        - [YouTube](https://www.youtube.com/c/DappLearning)
                        - [discord](https://discord.com/invite/gkENMS7cKz)
                        - [公众号](https://mp.weixin.qq.com/s/Y6fYmqrS9SpEw4Nd0ocUzg)
   - 国际化探索： 跟国外社区联系（web3university ,buildspace,......）

**Comment by liberhe** [comment]
电报群维护，国际化交流。

**Comment by York-Aaron-Swartz** [comment]


**Reply by qizhou** [reply]
欢迎欢迎，看看怎么样的合作形式呢？messari和dephi现在已经在做了。是不是可以看看其他的内容开始？

**Reply by York-Aaron-Swartz** [reply]
> 欢迎欢迎，看看怎么样的合作形式呢？messari和dephi现在已经在做了。是不是可以看看其他的内容开始？

我还在等我们负责人的看法以及回复，不过就我个人而言，我主要研究nft各个公链的标准协议，还有eth2.0或者叫模块化区块链，如果这些方向跟你们不冲突，我个人可以先参与，搞起来。

**Comment by DragonAgent** [comment]
6. 完成DAO组织建立与治理：
 -Snapshot提案及投票流程的建立
 -DAO NFT(水浒NFT)的投票权益，总量，分发机制，激励机制的确立
 -迁移未来提案至DAO，例如: 2023年规划

**Reply by yanyanho** [reply]
> 6. 完成DAO组织建立与治理：
>    -Snapshot提案及投票流程的建立
>    -DAO NFT(水浒NFT)的投票权益，总量，分发机制，激励机制的确立
>    -迁移未来提案至DAO，例如: 2023年规划

很棒，snapshot已经建立,https://snapshot.org/#/dapplearning.eth.   后续我们摸索中，来一起探索吧。
